pkg/controller/migstorage/validation.go

```diff
@@ -79,7 +67,7 @@ func (r ReconcileMigStorage) validateBackupStorage(storage *migapi.MigStorage) e
 			Status:   True,
 			Reason:   NotSet,
 			Category: Critical,
-			Message:  InvalidBSProviderMessage,
+			Message:  "spec.BackupStorageProvider empty",
```

Comment:
Let's not rewrite all of the messages, please.

Reply:
Maybe instead
```
`spec.backupStorageProvider` is empty. `spec.backupStorageProvider` must be: (aws|gcp|azure).
```
Some way of guiding the user towards the correct input would be good.